AlphaWallet/Browser/ViewModel/Dapps.swift

```diff
@@ -56,7 +56,11 @@ enum Dapps {
         Dapp(name: "Humanity", description: "Human Identity on Ethereum", url: "https://humanitydao.org", cat: "Social Media"),
         Dapp(name: "DEX.AG", description: "Trade cryptoassets at the best price", url: "https://dex.ag", cat: "Exchange"),
         Dapp(name: "Totle Swap", description: "Totle automatically finds and acquires the best price across decentralized exchanges for ERC-20 swaps", url: "https://swap.totle.com", cat: "Exchange"),
-        Dapp(name: "ATS Bridge", description: "ATS/ATS20 bridge for self transfers of ATS to ATS20", url: "https://bridge.artis.network/", cat: "Tool")
+        Dapp(name: "ATS Bridge", description: "ATS/ATS20 bridge for self transfers of ATS to ATS20", url: "https://bridge.artis.network/", cat: "Tool"),
+        Dapp(name: "dForce USDx", description: "A decentralized and synthetic indexed stablecoin with interest bearing capability", url: "https://usdx.dforce.network/", cat: "Finance"),
+        Dapp(name: "LENDF.ME", description: "Algorithmic money markets for dForce USDx", url: "https://www.lendf.me/", cat: "Finance")
+]
```

Comment:
This is breaking the build, please remove it and rebase your commits into one. Cheers